tox.ini

```diff
@@ -1,27 +1,25 @@
 [tox]
-envlist = {py36,py37,py38,py39}-{cov,nocov,diffcov}, qa, docs
+envlist = qa, docs, py{36,37,38,39}-{nocov,cov,diffcov}
```

Comment:
Faster tests should run earlier so we can quickly press Ctrl-C if we see `qa` or `docs` fail.